### PR TITLE
chapter 9: change Language

### PR DESCRIPTION
**Pull request by DyannT**
## Related Tickets:
- [#Ticket_ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by DyannT** [comment]
ready

**Comment by DyannT** [comment]
ready

File: resources/views/layouts/navigation.blade.php [hunk 1]
```diff
@@ -18,45 +18,81 @@
...
-                                :href="route('logout')">
-                                {{ __('Log Out') }}
+                        <x-slot name="content">
+                            <x-dropdown-link :href="route('locale',['lang' => 'en'])">
+                                {{ __('English') }}
                             </x-dropdown-link>
-                        </form>
-                    </x-slot>
-                </x-dropdown>
+                            <x-dropdown-link :href="route('locale',['lang' => 'vi'])">
```
Comment (ghost) [inline resources/views/layouts/navigation.blade.php]:
```suggestion
                            <x-dropdown-link :href="route('locale', ['lang' => 'vi'])">
```

File: resources/views/layouts/navigation.blade.php [hunk 2]
```diff
@@ -18,45 +18,81 @@
...
+
+                <div class="hidden sm:flex sm:items-center sm:ml-6">
+                    <x-dropdown align="right" width="48">
+                        <x-slot name="trigger">
+                            <button
+                                class="flex items-center text-sm font-medium text-gray-500 hover:text-gray-700 hover:border-gray-300 focus:outline-none focus:text-gray-700 focus:border-gray-300 transition duration-150 ease-in-out">
+                                <div class="flex items-center">
+                                    <div style="width: 40px; height: 40px; margin-right: 10px;">
+                                        <img style="border-radius: 50%;"
+                                             src="{{asset(!empty(Auth::user()->avatar) ? Auth::user()->avatar : 'images/avatar_default.png')}}"
```
Comment (ghost) [inline resources/views/layouts/navigation.blade.php]:
```suggestion
                                             src="{{ asset(!empty(Auth::user()->avatar) ? Auth::user()->avatar : 'images/avatar_default.png') }}"
```

File: resources/views/layouts/navigation.blade.php [hunk 3]
```diff
@@ -66,10 +102,10 @@
                     focus:text-gray-500 transition duration-150 ease-in-out">
                     <svg class="h-6 w-6" stroke="currentColor" fill="none" viewBox="0 0 24 24">
                         <path :class="{'hidden': open, 'inline-flex': ! open }" class="inline-flex"
-                             stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
-                            d="M4 6h16M4 12h16M4 18h16"/>
+                              stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+                              d="M4 6h16M4 12h16M4 18h16"/>
```
Comment (ghost) [inline resources/views/layouts/navigation.blade.php]: indent 4 space

File: resources/views/welcome.blade.php [hunk 4]
```diff
@@ -24,6 +24,8 @@
         <div class="relative flex items-top justify-center min-h-screen bg-gray-100 dark:bg-gray-900 sm:items-center py-4 sm:pt-0">
             @if (Route::has('login'))
                 <div class="hidden fixed top-0 right-0 px-6 py-4 sm:block">
+                    <a href="{{ route('locale',['lang' => 'en']) }}" class="text-sm text-gray-700 dark:text-gray-500 underline">{{__('English')}}</a>
+                    <a href="{{ route('locale',['lang' => 'vi']) }}" class="text-sm text-gray-700 dark:text-gray-500 underline">{{__('Vietnamese')}}</a>
```
Comment (ghost) [inline resources/views/welcome.blade.php]:
```suggestion
                    <a href="{{ route('locale', ['lang' => 'en']) }}" class="text-sm text-gray-700 dark:text-gray-500 underline">{{__('English')}}</a>
                    <a href="{{ route('locale', ['lang' => 'vi']) }}" class="text-sm text-gray-700 dark:text-gray-500 underline">{{__('Vietnamese')}}</a>
```

**Comment by DyannT** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**